diff -Naur Linux-PAM-1.3.0/modules/pam_console/Makefile.am Linux-PAM-1.3.0.tpg/modules/pam_console/Makefile.am
--- Linux-PAM-1.3.0/modules/pam_console/Makefile.am	2018-01-26 10:28:36.017629000 +0100
+++ Linux-PAM-1.3.0.tpg/modules/pam_console/Makefile.am	2018-01-26 10:31:20.974981732 +0100
@@ -51,6 +51,8 @@
 pam_console_la_CFLAGS = $(AM_CFLAGS)
 pam_console_apply_CFLAGS = $(AM_CFLAGS) @PIE_CFLAGS@
 
+configfile.tab.h: configfile.tab.c
+
 configfile.tab.c: configfile.y
 	$(YACC) $(BISON_OPTS) -o $@ -p _pc_yy $<
 	sh $(srcdir)/sed-static $@
