diff -up Linux-PAM-1.3.0/configure.ac.omv~ Linux-PAM-1.3.0/configure.ac
--- Linux-PAM-1.3.0/configure.ac.omv~	2018-05-09 18:18:45.672869056 +0200
+++ Linux-PAM-1.3.0/configure.ac	2018-05-09 18:20:12.816870163 +0200
@@ -561,7 +561,14 @@ else
      AC_PATH_PROG([BROWSER], [links])
      if test ! -z "$BROWSER"; then
           BROWSER="$BROWSER -no-numbering -no-references -dump"
-     else
+     fi
+     if test -z "$BROWSER"; then
+          AC_PATH_PROG([BROWSER], [elinks])
+     fi
+     if test -z "$BROWSER"; then
+          AC_PATH_PROG([BROWSER], [lynx])
+     fi
+     if test -z "$BROWSER"; then
           enable_docu=no
      fi
 fi
