--- Linux-PAM-1.1.0/modules/pam_console/pam_console_apply.c.error	2009-10-06 17:34:02.000000000 +0200
+++ Linux-PAM-1.1.0/modules/pam_console/pam_console_apply.c	2009-10-06 17:39:14.000000000 +0200
@@ -65,7 +65,7 @@ parse_files(void)
 	on system locale */
 	oldlocale = setlocale(LC_COLLATE, "C");
 
-	rc = glob(PERMS_GLOB, GLOB_NOCHECK, NULL, &globbuf);
+	rc = glob(PERMS_GLOB, 0, NULL, &globbuf);
 	setlocale(LC_COLLATE, oldlocale);
 	if (rc)
 		return;
